src/components/CR8 - GalleryText/GalleryText.vue

```diff
@@ -0,0 +1,247 @@
+<template>
+    <div class="gallery-text">
+        <!-- Title -->
+        <h2 class="title">Our Ecosystem</h2>
+
+        <!-- Subtitle w/ Arrow -->
+        <div class="subtitle">
+            <span class="subtitle-text">We develop and produce original content for all genres and formats.</span>
+            <span class="prompt">
+                <span class="text" >explore</span>
+                <svg-arrow-right class="arrow" />
+            </span>
+        </div>
+        
+        <!-- Block Links -->
+        <div class="text-block">
+            <span
+                v-for="company in companies"
+                :key="company.name"
+            >
+                <nuxt-link class="company-link" :to="`/${company.name}`" tag="a">
+                    <component 
+                        :is="`logo-${company.name}`"
+                        class="company-logo"
+                        @mouseover="set(company)" 
+                        @mouseout="set(null)" />
+                </nuxt-link>
+                <span class="text">{{company.text}} </span>
+            </span>
+        </div>
+
+        <!-- Images absolute -->
+        <transition
+            v-for="(image, index) in images"
```

Comment:
I think you are using this wrong, you don't put the `v-for` on the transition. You use it like this:

```
<transition-group 
            name="fade"
            mode="in-out"
>
            <wp-image
             v-for="(image, index) in images"
             :key="image.id"
                v-show="index == activeImage"
                class="image"
                :image="image"
                mode="fullbleed"
            />
</transition-group>
```

I'm pretty sure that is how you are supposed to do it.

Reply:
i changed the mode to "in-out" but the background text was still appearing significantly.  Created custom keyframe animation and added a delay to the "leave-action" to make sure the new image fades in before the current image fades out.